src/test/java/de/tum/in/www1/artemis/QuizExerciseIntegrationTest.java

```diff
@@ -1342,6 +1342,84 @@ public void testRedundantActionsBadRequest() throws Exception {
         request.putWithResponseBody("/api/quiz-exercises/" + quizExercise.getId() + "/lorem-ipsum", quizExercise, QuizExercise.class, HttpStatus.BAD_REQUEST);
     }
 
+    /**
+     * test that a quiz question with an explanation within valid length can be created
+     */
+    @Test
+    @WithMockUser(value = "instructor1", roles = "INSTRUCTOR")
+    public void testMultipleChoiceQuizExplanationLength_Valid() throws Exception {
+        int validityThreshold = 500;
+
+        QuizExercise quizExercise = createMultipleChoiceQuizExerciseDummy();
+        MultipleChoiceQuestion question = (MultipleChoiceQuestion) quizExercise.getQuizQuestions().get(0);
+        question.setExplanation("0".repeat(validityThreshold));
+
+        request.postWithResponseBody("/api/quiz-exercises/", quizExercise, QuizExercise.class, HttpStatus.CREATED);
```

Comment:
please assert that body is not null, if it is successful call.

Reply:
Done